Power BI Tutorial .Rmd

```diff
@@ -0,0 +1,115 @@
+---
+title: "Power BI Tutorial"
+author: "Ellen Chen"
+date: "3/10/2021"
+output:
+  html_document: default
+  pdf_document: default
+---
+
+## Quickstart Tutorial to Microsoft Power BI 
```

Comment:
As per 2.2.3 (https://jtr13.github.io/cc21/github-submission-instructions.html), on the first line of your document, enter a single hashtag (#), followed by a single white space, and then your title.